content/docs/command-reference/import-url.md

```diff
@@ -29,7 +29,7 @@ external data source changes. Example scenarios:
 - A shared dataset on a remote storage that is managed and updated outside DVC.
 
 > Note that `dvc get-url` corresponds to the first step this command performs
-> (just download the file or directory).
+> (downloads the file or directory).
```

Comment:
"Just" is also OK here: Not condescending but descriptive.

But that s was missing indeed, so pls change back to `just downloads`.

Reply:
Can we find a better word for just in this scenario?
Alex.js warning:

 warning  `just` may be insensitive, try not to use it            just      retext-equality

Reply:
What is Alex.js ?

> warning just may be insensitive

It's not insensitive here. It means "downloads only" but sounds more natural. Please *just* roll it back :P